controllers/tests/controller_server/host_definer/settings.py

```diff
@@ -0,0 +1,69 @@
+
+NAME = 'name'
+NAMESPACE = 'namespace'
+SPEC = 'spec'
+METADATA = 'metadata'
+STATUS = 'status'
+DRIVERS = 'drivers'
+NODE_ID_FIELD_IN_CSI_NODE = 'nodeID'
+CSI_PROVISIONER_NAME = 'block.csi.ibm.com'
+FAKE_SECRET = 'fake_secret'
+FAKE_SECRET_NAMESPACE = 'fake_secret_namespace'
+FAKE_SECRET_ID = (FAKE_SECRET, FAKE_SECRET_NAMESPACE)
+FAKE_NODE_NAME = 'fake_node_name'
+FAKE_DAEMON_SET_NAME = 'fake_daemon_set_name'
+FAKE_NODE_PODS_NAME = '{}-jdf5g'.format(FAKE_DAEMON_SET_NAME)
+FAKE_SECRET_ARRAY = 'management_address'
+FAKE_SECRET_PASSWORD = 'fake_password'
+FAKE_SECRET_USER_NAME = 'fake_user_name'
+FAKE_STORAGE_CLASS = 'fake_storage_class'
+IQN = 'iqn.1994-05.com.redhat:686358c930fe'
+FAKE_NODE_ID = '{};;;{}'.format(IQN, HOST_NAME)
+FAKE_CSI_PROVISIONER = 'fake_csi_provisioner'
+TRUE_STRING = 'true'
+LABELS = 'labels'
+FAKE_LABEL = 'FAKE_LABEL'
+MANAGE_NODE_LABEL = 'hostdefiner.block.csi.ibm.com/manage-node'
+FORBID_DELETION_LABEL = 'hostdefiner.block.csi.ibm.com/do-not-delete-definition'
+WATCHER_HELPER_PATH = 'controllers.servers.host_definer.watcher.watcher_helper'
+NODES_WATCHER_PATH = 'controllers.servers.host_definer.watcher.node_watcher'
+SECRET_WATCHER_PATH = 'controllers.servers.host_definer.watcher.secret_watcher'
+CSI_NODE_WATCHER_PATH = 'controllers.servers.host_definer.watcher.csi_node_watcher'
+STORAGE_CLASS_WATCHER_PATH = 'controllers.servers.host_definer.watcher.storage_class_watcher'
+SETTINGS_PATH = 'controllers.servers.host_definer.settings'
+RESOURCE_VERSION = 'resource_version'
+FAKE_RESOURCE_VERSION = '495873498573'
+FAKE_UID = '50345093486093'
+TYPE = 'type'
+OBJECT = 'object'
+KUBERNETES_MANAGER_INIT_FUNCTIONS_TO_PATCH = ['_load_cluster_configuration', '_get_dynamic_client']
+UPDATED_PODS = 'updated_number_scheduled'
+NODE_NAME_FIELD_IN_PODS = 'node_name'
+DESIRED_UPDATED_PODS = 'desired_number_scheduled'
+DELETED_EVENT_TYPE = 'DELETED'
+MODIFIED_EVENT_TYPE = 'MODIFIED'
+ADDED_EVENT = 'ADDED'
+UID = 'uid'
+PHASE = 'phase'
+READY_PHASE = 'Ready'
+HOST_DEFINITION_FIELD = 'hostDefinition'
+SECRET_NAME_FIELD = 'secretName'
+SECRET_NAMESPACE_FIELD = 'secretNamespace'
+NODE_NAME_FIELD_HOST_DEFINITION = 'nodeName'
+NODE_ID_FIELD_IN_HOST_DEFINITION = 'nodeId'
+DATA = 'data'
+FAIL_MESSAGE_FROM_STORAGE = 'fail_from_storage'
+PENDING_CREATION_PHASE = 'PendingCreation'
+PENDING_DELETION_PHASE = 'PendingDeletion'
+SUCCESS_MESSAGE = 'Host defined successfully on the array'
+HOST_DEFINITION_PENDING_VARS = {'HOST_DEFINITION_PENDING_RETRIES': 3,
+                                'HOST_DEFINITION_PENDING_EXPONENTIAL_BACKOFF_IN_SECONDS': 0.2,
+                                'HOST_DEFINITION_PENDING_DELAY_IN_SECONDS': 0.2}
+PROVISIONER_FIELD = 'provisioner'
+PARAMETERS_FIELD = 'parameters'
+STORAGE_CLASS_SECRET_FIELD = 'csi.storage.k8s.io/secret-name'
+STORAGE_CLASS_SECRET_NAMESPACE_FIELD = 'csi.storage.k8s.io/secret-namespace'
+CSI_IBM_GROUP = 'csi.ibm.com'
+VERSION = 'v1'
+HOST_DEFINITION_PLURAL = 'hostdefinitions'
```

Comment:
why [again](https://github.com/IBM/ibm-block-csi-driver/pull/579/files#diff-aaed924057203f2c79d77261ef8f65c796ce39d3e165c961998a2db796550fa8R7)?

Reply:
I am not sure where the link is directed to, is directed to a file

Reply:
we have `HOST_DEFINITION_PLURAL = 'hostdefinitions'` twice in the code

Reply:
so you want me to call the settings in the code from the unit tests?

Reply:
you can move it to controllers/common and then its ok..

Reply:
done for some of them, I opened a CSI-4873 ticket to complete it